backoffice/backoffice/workflows/constants.py

```diff
@@ -33,6 +33,10 @@ class WorkflowType(models.TextChoices):
 class ResolutionDags(models.TextChoices):
     accept = "accept", "author_create_approved_dag"
     reject = "reject", "author_create_rejected_dag"
+    accept_curate = "accept_curate", "author_create_approved_dag"
+
+
+DECISION_CHOICES = ResolutionDags.choices + []
```

Comment:
why `+ []`?

Reply:
It was so I would remember to add the decisions regarding other stuff in the future. Remove